changelog.txt

```diff
@@ -10,6 +10,8 @@
 * Fix - Prevent Payment Request buttons from showing up in Composite Product pages.
 * Update - Updated @woocommerce/experimental package to v2.1.0.
 * Add - Onboarding flows on the admin WooCommerce > Subscriptions screen for stores with no subscriptions yet.
+* Add - IPP Styled Receipts: Merchant brand settings (API).
+* Add - IPP Styled Receipts: Card readers page.
```

Comment:
I think a more appropriate message would be "Add - Card Reader receipt settings page," since the Card Readers page itself is not yet ready.

Reply:
+1 for updating the change log entry

Reply:
Updated.